test_runner/performance/test_perf_olap.py

```diff
@@ -16,6 +16,21 @@ class LabelledQuery:
     label: str
     query: str
 
+# create extension pg_stat_statements before all tests in this module if it does not exist
+# and TEST_OLAP_COLLECT_PG_STAT_STATEMENTS is set to true (default false)
+# Theoretically this could be in a module or session scope fixture,
+# however the code depends on other fixtures that have function scope
+@pytest.mark.remote_cluster
+def test_clickbench_create_pg_stat_statements(remote_compare: RemoteCompare):
```

Comment:
I would add that we rely on this test to be executed first to a comment

Reply:
I had "before all tests" but I will emphasize it more